ko/sql/schema/index_stmt.rst

```diff
@@ -336,8 +336,7 @@ ALTER INDEX
     *   CUBRID 10.0 버전부터는 테이블 이름을 생략할 수 없다.
     
     *   CUBRID 10.0 버전부터는 테이블 이름 뒤에 칼럼 이름을 추가하더라도 이는 무시되며, 예전 인덱스와 동일한 칼럼으로 재생성된다.
-
-    *   prefix 인덱스 기능은 제거될 예정(deprecated)이므로, 더 이상 사용을 권장하지 않는다.
```

Comment:
why remove the line ?

Reply:
This is mentioned in “NOTE” in the “CREATE INDEX” description.
Also, there is no need to mention the prefix index here since the ALTER INDEX syntax cannot change the configuration.

Reply:
OK, i see.